draft-ietf-mls-protocol.md

```diff
@@ -3903,11 +3920,7 @@ new members need information to bootstrap their local group state.
 ~~~ tls
 struct {
     CipherSuite cipher_suite;
-    opaque group_id<V>;
-    uint64 epoch;
-    opaque tree_hash<V>;
-    opaque confirmed_transcript_hash<V>;
-    Extension group_context_extensions<V>;
```

Comment:
- [ ] Please also update GroupInfoTBS below.
- [ ] Three references to `group_context_extensions` in text